packages/slate-react/src/components/editable.tsx

```diff
@@ -205,10 +205,14 @@ export const Editable = (props: EditableProps) => {
         )
       }
       const leafEl = newDomRange.startContainer.parentElement!
+      leafEl.getBoundingClientRect = newDomRange.getBoundingClientRect.bind(
+        newDomRange
+      )
       scrollIntoView(leafEl, {
         scrollMode: 'if-needed',
         boundary: el,
       })
+      delete leafEl.getBoundingClientRect
```

Comment:
🤔 this throws a typescript error could you elaborate why this is needed @gztomas?

Reply:
@juliankrispel this restores the override happening at line `208`, so any future call to `leafEl.getBoundingClientRect` will get the native `getBoundingClientRect` implementation from `leafEl` prototype. This is for cleanup purposes, as this code doesn't know who else might need to call `getBoundingClientRect` for this element. I guess TS can't know that even deleting this, `getBoundingClientRect ` will still be available, from the prototype.

Reply:
thanks for explaining @gztomas